template-parts/content-none.php

```diff
@@ -15,10 +15,10 @@
 	</header><!-- .page-header -->
 
 	<div class="page-content">
-		<?php
-		if ( is_home() && current_user_can( 'publish_posts' ) ) : ?>
+		<?php if ( is_home() && current_user_can( 'publish_posts' ) ) : ?>
 
-			<p><?php
+			<p>
+			<?php
```

Comment:
In this case, I'd consider adding the `<p>/</p>` tags to the `printf()` call instead of going in and out of PHP.